Makefile

```diff
@@ -70,6 +70,9 @@ minio:
 	git submodule update --init minio
 	cd minio && $(MAKE)
 
+update-external-deps:
+	curl -L -o src/generate_list.sh https://raw.githubusercontent.com/keylime/keylime/master/scripts/create_whitelist.sh
```

Comment:
If this is still at the POC stage, I'd just drop this entirely and do what Micah suggested re. commenting clearly at the top of the script where it came from. I think later on, we'll want to do this more natively as mentioned earlier, which would drop the dep on the script.

Reply:
The script the does the hashing in a booted OS isn't a POC and comes from the official upstream. However, moving over to the `ostree` native way this likely is no longer needed. For now the `Makefile` helps update this script when needed and, if required, I could modify it there by injecting a comment.